### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by peter7z**
This solves a problem with tests and adds a test for signup and home pages.

### CODE REVIEW

File: src/containers/HomePage.spec.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { fromJS } from 'immutable';
+// import nock from 'nock';
```
Comment (felipegarcia92) [inline src/containers/HomePage.spec.js]: Remove commented code

File: src/containers/HomePage.spec.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    const location = {
+      pathname: '/',
+      search: '/',
+      hash: '',
+      key: '9kzkus'
+    };
+    subject = mount(withStore(<HomePage location={location} />, store, location));
+  });
+
+  it('should render a div', () => {
```
Comment (felipegarcia92) [inline src/containers/HomePage.spec.js]: `a div`? There are many divs, maybe you should specify that you're looking for an specific root div tag.

File: src/containers/HomePage.spec.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { fromJS } from 'immutable';
+// import nock from 'nock';
+
+import HomePage from './HomePage';
+import configureStore from '../store/configureStore';
+import { withStore } from '../utils/testHelpers';
+
+describe('<HomePage />', () => {
```
Comment (felipegarcia92) [inline src/containers/HomePage.spec.js:11]: This spec is a little vague. What do we expect to see when arriving to the homepage? One left menu component and a map component, right? Can you test on the presence of those?

File: src/containers/SignUpPage.spec.js [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+        },
+        small_thumb: {
+          url: 'http://urlexample.com/avatar/12345.jpg'
+        }
+      }
+    };
+
+    sessionService.saveUser = jest.fn(() => Promise.resolve());
+    sessionService.saveSession = jest.fn(() => Promise.resolve());
+    // sessionService.loadSession = jest.fn(() => Promise.reject('Session not found'));
```
Comment (felipegarcia92) [inline src/containers/SignUpPage.spec.js]: Remove commented code

File: src/containers/SignUpPage.spec.js [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+      nock(process.env.API_URL)
+        .post('//users', humps.decamelizeKeys(fakeAccount))
+        .reply(200, humps.camelizeKeys({ fakeAccount: fakeAccountResponse }));
+
+      username.simulate('change', { target: { value: fakeAccount.user.username } });
+      email.simulate('change', { target: { value: fakeAccount.user.email } });
+      password.simulate('change', { target: { value: fakeAccount.user.password } });
+      passwordConfirmation.simulate('change', { target: { value: fakeAccount.user.password } });
+      gender.simulate('change', { target: { value: fakeAccount.user.gender } });
+      const inp = gender.find('input').at(0);
```
Comment (felipegarcia92) [inline src/containers/SignUpPage.spec.js]: `inp` doesn't say much. `genderInput` maybe?  `genderSelect`?

**Review by felipegarcia92 — CHANGES_REQUESTED** [review]
Good job on the tests, specially the SignUpPage ones 👍

**Review by felipegarcia92 — APPROVED** [review]
👌

**Review by juanlacu — APPROVED** [review]
Good work on this one!

**Pull request merged**